WordPress/src/main/java/org/wordpress/android/ui/main/jetpack/migration/compose/state/WelcomeStep.kt

```diff
@@ -179,19 +200,31 @@ val previewSiteListItems = mutableListOf<SiteListItemUiState>().apply {
     }
 }
 
+private val previewUiState = UiState.Content.Welcome(
+        sites = previewSiteListItems,
+        primaryActionButton = WelcomePrimaryButton {},
+        secondaryActionButton = WelcomeSecondaryButton {},
+)
+
+@Preview(showBackground = true, widthDp = 414, heightDp = 897)
+@Preview(showBackground = true, widthDp = 414, heightDp = 897, uiMode = Configuration.UI_MODE_NIGHT_YES)
```

Comment:
Not suggesting a change here, but curious if you considered using a `Device` from `androidx.compose.ui:ui-tooling-preview`?

Reply:
> Not suggesting a change here, but curious if you considered using a `Device` from `androidx.compose.ui:ui-tooling-preview`?

Thank you for this remark @mkevins 🙇 
I started development on this and wanted to have the exact screen size from the Figma designs as a starting point, but the design screens are for iOS screen size.
It makes sense to replace this with a preset preview device, which has already been done for the other steps in the flow 👍

Reply:
Addressed in commit: 2116a45